src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write code here
+const titles = document.querySelector('tr');
+const tbody = document.querySelector('tbody');
+const rows = tbody.querySelectorAll('tr');
+
+const namesSorted = [...rows].sort((name1, name2) =>
+  name1.firstElementChild.textContent.localeCompare(
+    name2.firstElementChild.textContent));
+
+const positionSorted = [...rows].sort((position1, position2) =>
+  position1.firstElementChild.nextElementSibling.textContent.localeCompare(
+    position2.firstElementChild.nextElementSibling.textContent));
+
+const ageSorted = [...rows].sort((value1, value2) =>
+  value1.lastElementChild.previousElementSibling.textContent
+  - value2.lastElementChild.previousElementSibling.textContent);
+
+const salarySorted = [...rows].sort((value1, value2) =>
+  value1.lastElementChild.textContent.replaceAll(',', '').slice(1)
+  - value2.lastElementChild.textContent.replaceAll(',', '').slice(1));
+
+function addSortedList(sorted) {
+  tbody.innerHTML = `
+    ${[...sorted].map(person => `
+    <tr>${person.innerHTML}</tr>
+    `).join('')}`;
+};
```

Comment:
try to use like that
In this case you sort onClick
in your case - you sort four time
```suggestion
const namesSorted = (name1, name2) =>
  name1.firstElementChild.textContent.localeCompare(
    name2.firstElementChild.textContent));

const positionSorted = (position1, position2) =>
  position1.firstElementChild.nextElementSibling.textContent.localeCompare(
    position2.firstElementChild.nextElementSibling.textContent));

const ageSorted = (value1, value2) =>
  value1.lastElementChild.previousElementSibling.textContent
  - value2.lastElementChild.previousElementSibling.textContent);

const salarySorted = (value1, value2) =>
  value1.lastElementChild.textContent.replaceAll(',', '').slice(1)
  - value2.lastElementChild.textContent.replaceAll(',', '').slice(1));

function addSortedList(callback) {
const sorted = [...rows].sort(callback);

  tbody.innerHTML = `
    ${sorted.map(person => `
    <tr>${person.innerHTML}</tr>
    `).join('')}`;
};
```